examples/contracts/src/lib.rs

```diff
@@ -636,7 +643,7 @@ mod tests {
             .await?;
         // ANCHOR_END: multi_call_cost_estimation
 
-        let expected_gas = 4402;
+        let expected_gas = 11231;
```

Comment:
We should probably mention in the release notes somewhere, and maybe the pub docs of `TransactionCost` that the `gas_used` is now the total gas, including both script gas and regular gas.

Otherwise, someone might get surprised by the bump.

Also, since we're reporting details such as `metered_bytes_size`, we might include the script gas used as well, since we're loosing that from `TransactionCost` with this pr.

Reply:
Thanks makes sense.